assets/stylesheets/main-responsive.css

```diff
@@ -148,7 +148,7 @@
 	#flowchart-mobile {
 		display: none
 	}
-	#flowchart {
+	#services {
```

Comment:
Since this display: block used as default, we may have just one default setting and override in small screen size.

Reply:
removed unnecessary display: blocks